File: QTD2.Infrastructure/Model/TrainingIssue/TrainingIssueOverview_VM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QTD2.Infrastructure.Model.TrainingIssue
{
   public class TrainingIssueOverview_VM
    {
        public int TrainingIssuesOpen { get; set; }
        public int TrainingIssuesClosed { get; set; }
        public int TrainingIssuesWithPendingActionItems { get; set; }
        public int TrainingIssuesWithNoActionItems { get; set; }
        public List<TrainingIssueOverview_TrainingIssue_VM> TrainingIssues { get; set; } = new List<TrainingIssueOverview_TrainingIssue_VM>();
        public TrainingIssueOverview_VM()
        {

        }

        public TrainingIssueOverview_VM(int trainingIssuesOpen,int trainingIssuesClosed, int trainingIssuesWithPendingActionItems, int trainingIssuesWithNoActionItems)
        {
            TrainingIssuesOpen = trainingIssuesOpen;
            TrainingIssuesClosed = trainingIssuesClosed;
            TrainingIssuesWithPendingActionItems = trainingIssuesWithPendingActionItems;
            TrainingIssuesWithNoActionItems = trainingIssuesWithNoActionItems;
        }
    }
}

File: QTD.SPA/src/app/_DtoModels/TrainingIssues/TrainingIssueOverview_VM.ts
import { TrainingIssueOverview_TrainingIssue_VM } from "./TrainingIssueOverview_TrainingIssue_VM";

export class TrainingIssueOverview_VM {
  trainingIssuesOpen: number;
  trainingIssuesClosed: number;
  trainingIssuesWithPendingActionItems: number;
  trainingIssuesWithNoActionItems: number;
  trainingIssues: TrainingIssueOverview_TrainingIssue_VM[];
}
